docs/reference/tab-widgets/troubleshooting/data/increase-node-capacity.asciidoc

```diff
@@ -0,0 +1,251 @@
+of data nodes in the {es} cluster so that the replica shards can be assigned. This can be done by
+editing your deployment. But first you need to discover which tier an index is targeting for assignment.
+Do this using {kib}.
+
+**Use {kib}**
+
+//tag::kibana-api-ex-1[]
+. Log in to the {ess-console}[{ecloud} console].
++
+
+. On the **Elasticsearch Service** panel, click the name of your deployment.
++
+
+NOTE:
+If the name of your deployment is disabled your {kib} instances might be
+unhealthy, in which case please contact https://support.elastic.co[Elastic Support].
+If your deployment doesn't include {kib}, all you need to do is
+{cloud}/ec-access-kibana.html[enable it first].
+
+. Open your deployment's side navigation menu (placed under the Elastic logo in the upper left corner)
+and go to **Dev Tools > Console**.
++
+[role="screenshot"]
+image::images/kibana-console.png[{kib} Console,align="center"]
+
+To inspect which tier an index is targeting for assignment, use the <<indices-get-settings, get index setting>>
+API to retrieve the configured value for the `index.routing.allocation.include._tier_preference`
+setting:
+
+[source,console]
+----
+GET /my-index-000001/_settings/index.routing.allocation.include._tier_preference?flat_settings
+----
+// TESTRESPONSE[skip:the result is for illustrating purposes only as don't want to change a cluster-wide setting]
+
+
+The response will look like this:
+
+[source,console-result]
+----
+{
+  "my-index-000001": {
+    "settings": {
+      "index.routing.allocation.include._tier_preference": "data_warm,data_hot" <1>
+    }
+  }
+}
+----
+// TESTRESPONSE[skip:the result is for illustrating purposes only]
+
+<1> Represents a comma separated list of data tier node roles this index is allowed
+to be allocated on, the first one in the list being the one with the higher priority
+i.e. the tier the index is targeting.
+e.g. in this example the tier preference is `data_warm,data_hot` so the index is
+targeting the `warm` tier and more nodes with the `data_warm` role are needed in
+the {es} cluster.
+
+//end::kibana-api-ex-2[]
+
+//tag::increase-azs[]
+[increase-azs]
+Now that you know the tier, increase the number of availability zones.
+
+. Go back to the deployment's landing page by clicking on the three horizontal bars on the top left of the screen
+and choosing **Manage this deployment**. On that page click the **Manage** button, and choose **Edit deployment**.
+
+. In the {es} section, find the tier where the replica shards could not be assigned. Select a higher number
+of availability zones for that tier.
++
+
+[role="screenshot"]
+image::images/data-tiers/ess-advanced-config-data-tiers.png[{kib} Console,align="center"]
+//end::increase-azs[]
+
+If it is not possible to increase the number of availability zones, you can reduce the number of replicas of your index
+data. We'll achieve this by inspecting the <<dynamic-index-number-of-replicas,`index.number_of_replicas`>> index setting
```

Comment:
I found it's a bit strange to talk about increasing the availabitiy zone which is related to fault tolerance.  Shouldn't we be talking about adjusting the `size per zone` from the dropdown list? 

We can also link to this ESS page: https://www.elastic.co/guide/en/cloud/current/ec-customize-deployment-components.html#ec-cluster-size

Reply:
Having thought about it more, adding node capacity is related to the max size of nodes in the tier. 
--> If max size of the node in the tier >= 58GB, users can use either dropdown `size per zone` or `availability zone` to add more nodes.
--> If max size of the node < 58GB, users should use `availability zone` to add more nodes.

Reply:
Are you saying that in cloud we use at most 58 GB boxes, so if you ask for more than 58 GB of RAM you're implicitly asking for more than one node? That seems like an implementation detail that could change at any moment doesn't it?

Reply:
Maybe you're saying something else though? I just changed my "size per zone" to 64 GB, with 2 zones. But when I use `_cat/nodes`, I still only have 2 nodes. Is there an option I'm missing?

Reply:
I've modified my comment above to adjust the threshold to 64GB for all regions and cloud providers. I don't think this value is likely to change as it is very much depends on the [magical 32GB heap boundary](https://www.elastic.co/guide/en/elasticsearch/guide/current/heap-sizing.html#compressed_oops)

Reply:
OK I see what you're saying now. At 64 GB RAM per zone, you still only get 1 node/box, but every 64 GB you go up from there you get another node. I confirmed it in the _cat/nodes API.

Reply:
Let me know what you think of the way I phrased this. Maybe I should add something like "figure out which way is cheapest for you, and do that." I didn't want to put all of that math into this document though.